client/my-sites/domains/domain-management/transfer/transfer-to-other-site/confirmation-dialog.jsx

```diff
@@ -11,25 +11,80 @@ import { get } from 'lodash';
 /**
  * Internal Dependencies
  */
-import { getSite } from 'calypso/state/sites/selectors';
+import { currentUserHasFlag } from 'calypso/state/current-user/selectors';
+import { getSite, isCurrentPlanPaid } from 'calypso/state/sites/selectors';
 import { Dialog } from '@automattic/components';
+import { NON_PRIMARY_DOMAINS_TO_FREE_USERS } from 'calypso/state/current-user/constants';
 
 class TransferConfirmationDialog extends React.PureComponent {
 	static propTypes = {
+		isMapping: PropTypes.bool.isRequired,
 		isVisible: PropTypes.bool.isRequired,
 		targetSiteId: PropTypes.number.isRequired,
 		disableDialogButtons: PropTypes.bool.isRequired,
 		domainName: PropTypes.string.isRequired,
 		onConfirmTransfer: PropTypes.func.isRequired,
 		onClose: PropTypes.func.isRequired,
+		primaryWithPlansOnly: PropTypes.bool,
 	};
 
 	onConfirm = ( closeDialog ) => {
 		this.props.onConfirmTransfer( this.props.targetSite, closeDialog );
 	};
 
+	getMessage() {
+		const { domainName, isMapping, targetSite, translate } = this.props;
+		const targetSiteTitle = get( targetSite, 'title', translate( 'Site Title' ) );
+		if ( isMapping ) {
+			return translate(
+				'Do you want to transfer mapping of {{strong}}%(domainName)s{{/strong}} ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Do you want to transfer {{strong}}%(domainName)s{{/strong}} to site {{strong}}%(targetSiteTitle)s{{/strong}}?' )` *ES Score: 8*